Json/WorldToMap/WorldMapArea.json

```diff
@@ -59,6 +59,18 @@
     "UIMapId": 947,
     "Continent": "Azeroth"
   },
+  {
+    "ID": 15,
+    "MapID": 0,
+    "AreaID": 36,
+    "AreaName": "Alterac",
+    "LocLeft": 783.3333,
+    "LocRight": -2016.6666,
+    "LocTop": 1500.0,
+    "LocBottom": -366.66666,
+    "UIMapId": 1415,
+    "Continent": "Azeroth"
+  },
```

Comment:
im seeing you have been added a duplicate for the `Alterac` with the only difference `"UIMapId": 1415,` changed. 

Question does `Alterac` has two maps `1415` and `1416` ?
or
The `1415` is not the correct one and supposed to be `1416`?

Reply:
it has two, or even more.

Reply:
Do you have any specific `UIMapId` in that regard beside the mentioned ones?

`UIMapId` of 1415 means Azeroth=Eastern Kingdoms. So in other zones suffering the same issue, then they are not gonna work at all since 1415 is now bound to `Alterac` zone bounds.

Reply:
I am not sure, but I guess that this JSON was exported from mpq from another client version.

Reply:
I see that build is 2.4.3, but I used it for wow classic.

Reply:
have you spotted any difference in the `1.13.7` vs `2.4.3` dataset from the `uimap` and `worldmaparea` when focusing `Alterac` ?